pennylane/qchem/tapering.py

```diff
@@ -17,8 +17,9 @@
 # pylint: disable=unnecessary-lambda
```

Comment:
This is a general function that can in principle be used for any operation. I do not think it should be inside qchem, but rather be more visibly to general users of pennylane. The same applies to basically all tapering functionality, except I believe `taper_hf`. Let's consider doing this at some point

Reply:
The `taper` function is accessible as `qml.taper`, we can do the same for `taper_operation` if it is not qchem-specific anymore.

Reply:
Yes, let's do that. As far as I can tell, this method in principle applies to any operation

Reply:
Added `taper_operation` to __init__.